Source/compositorclient/RPI/ModeSet.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <stdint.h>
+
+class ModeSet
+{
+    public:
+        ModeSet(const ModeSet&) = delete;
+        ModeSet& operator= (const ModeSet&) = delete;
+
+        ModeSet();
+        ~ModeSet();
+
+    public:
+        const struct gbm_device* const UnderlyingHandle() const
+        {
+            return _device;
+        }
+        uint32_t Width() const;
+        uint32_t Height() const;
+        struct gbm_surface* CreateRenderTarget(uint32_t width, uint32_t height);
+        void DestroyRenderTarget(struct gbm_surface* surface);
+
+    private:
+        int  Open();
```

Comment:
uint32_t?

Reply:
It is a file descriptor from the OS wich, on failure is -1, so I guess it has to be int.